subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/core_dependency_management.adoc

```diff
@@ -14,36 +15,52 @@ For an overview of dependency management terms, see <<dependency_management_term
 [[sec:dependency-mgmt-in-gradle]]
 == Dependency Management in Gradle
 
-[.inset]
+.Dependencies management at a glance
 image::dependency-management-resolution.png[]
 
 Gradle has built-in support for dependency management.
 
-We’ll explore the main concepts with the help of the theoretical but common project to define terms:
+Let's explore the main concepts with the help of a theoretical but common project:
 
 * This project builds Java source code.
-* Some of the Java source files import classes from link:https://github.com/google/guava[Google Guava], a open-source library.
+* Some Java source files import classes from the link:https://github.com/google/guava[Google Guava] library.
 * This project uses link:http://junit.org/junit5/[JUnit] for testing.
 
-This project uses Guava and JUnit as _dependencies_.
-Gradle fetches dependencies from _repositories_.
-You can <<declaring_repositories.adoc#declaring-repositories,declare repositories>> to tell Gradle where to fetch dependencies.
+The `build.gradle` file might look as follows:
+
+.Gradle build file with dependencies
+====
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/kotlin",files="build.gradle.kts"]
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/groovy",files="build.gradle"]
+====
+<1> Define artifact sources for project
+<2> Remote and local repositories for artifact locations (link:https://mvnrepository.com/repos/central[Maven] and link:https://maven.google.com/[Google] repositories)
+<3> Define artifacts used by project
+<4> Specific artifact name and version within a specific scope (Guava 31.1-jre for implementation and JUnit 4.13.2 for testImplementation)
+
+As we can see, this theoretical project uses Guava and JUnit as _dependencies_.
+
+Gradle fetches dependencies from the specified _repositories_, in this case, link:https://mvnrepository.com/repos/central[Maven] and link:https://maven.google.com/[Google].
+You can <<declaring_repositories.adoc#declaring-repositories,declare repositories>> to tell Gradle where to fetch dependencies locally or remotely.
+
 Repositories offer dependencies in multiple formats.
 For information about the formats supported by Gradle, see <<declaring_dependencies.adoc#sec:dependency-types,dependency types>>.
 
-A build script developer can <<declaring_dependencies.adoc#declaring-dependencies,declare dependencies>> for different scopes.
-For example, Gradle could use one scope to compile source code and another to execute tests.
+A build engineer can <<declaring_dependencies.adoc#declaring-dependencies,declare dependencies>> for different scopes.
+In the example project, the JUnit was declared only for the `testImplementation` scope which is used only when testing and not at runtime.
+As such, one scope is used to compile source code and another to execute tests.
```

Comment:
I think this sentence requires too much implied knowledge to make sense. The previous one was the same tbh.

Maybe we should replace this with a mention that Gradle uses configurations to scope dependencies and link to the relevant section. Something like [this](https://docs.gradle.org/current/userguide/declaring_dependencies.html#sec:choosing-configuration).

And then probably use the _configuration_ term in the first sentence of that paragraph, in place of _scopes_.

Reply:
I made some changes, let me know what you think of the latest commit.